diff --git a/includes/common.inc b/includes/common.inc
index 339a69b..2e78af1 100644
--- a/includes/common.inc
+++ b/includes/common.inc
@@ -2305,6 +2305,7 @@ function url($path = NULL, array $options = array()) {
   }
   elseif (!empty($path) && !$options['alias']) {
     $language = isset($options['language']) && isset($options['language']->language) ? $options['language']->language : '';
+    require_once DRUPAL_ROOT . '/includes/path.inc';
     $alias = drupal_get_path_alias($original_path, $language);
     if ($alias != $original_path) {
       $path = $alias;
